src/backend/utils/analytics.ts

```diff
@@ -0,0 +1,23 @@
+// todo: remove, deprecated
+
+export default class AnalyticsService {
+  private GA;
+
+  constructor() {
+    if (window.hasOwnProperty('ga')) {
+
+      // todo: figure out what rule to specifically ignore on next line here
+      // instead using type assertion here to get around issue
+      // https://basarat.gitbooks.io/typescript/content/docs/types/type-assertion.html
+      this.GA = (window as any).ga;
+    }
```

Comment:
Move all the calls https://github.com/rangle/augury/pull/1094/files#diff-0fa1d9c4e14a9a44bcde198728d4435eR22 here you should need `analytics.ts`